packages/relay/src/routes/post.ts

```diff
@@ -137,13 +137,46 @@ async function createPost(
             calldata
         )
 
-        const post = await db.create('Post', {
-            content: content,
-            cid: cid,
-            epochKey: epochKeyProof.epochKey.toString(),
-            epoch: epoch,
-            transactionHash: hash,
-            status: 0,
+        const epochKey = epochKeyProof.epochKey.toString()
+
+        // after post data store in DB, should add 1 to epoch key counter
+        await db.transaction(async (txDB) => {
```

Comment:
Should we use functional args to make this method become general, since it will be used in vote and comment also?
```
async function addActionCount(db: DB, action: (TransactionDB) => undefined) { 
   
   db.transaciton( async (txDB) => {
         action(txDB)  <--- make it become variable
         
         counter .... 
   
   }



 }
```

Reply:
That's a great idea, I move the counter logic out to an util function as you described.